server-src/src/main/java/com/itt/Application.java

```diff
@@ -61,7 +61,7 @@ public final void run(final String... args)
     /**
      * The bean method that lists all the registered beans. uncomment Bean
      * annotation if you want to list the registered beans.
-     * 
+     *
```

Comment:
I do not see any changes in this file, please remove from snapshot

Reply:
removed unnecessary space.